MANIFEST.in

```diff
@@ -1,3 +1,5 @@
 include LICENSE.txt
 include requirements.txt
 include qiskit_experiments/VERSION.txt
+include qiskit_experiments/library/randomized_benchmarking/data/*.npz
```

Comment:
This is because `data` doesn't have init file and thus it's not recognized?

Reply:
Resolved in a04310a with above.